srcpkgs/eiwd/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'eiwd'
+pkgname=eiwd
+version=1.6r1
+revision=1
+wrksrc="${pkgname}-${version/r/-}"
+build_style=gnu-configure
+configure_args="--disable-systemd-service --enable-pie
+ --disable-external-ell --disable-dbus --enable-wired"
+hostmakedepends="python3-docutils pkg-config automake libtool"
+makedepends="readline-devel ell-devel>=0.31"
+depends="ell>=0.31"
+checkdepends="python3"
+short_desc="""iwd without dbus"
```

Comment:
bonus ""s?

Reply:
xlint workaround for lowercase initial... open to suggestions